Team-C/test_wizard/lib/views/modify_test_view.dart

```diff
@@ -94,6 +93,96 @@ class ColumnHeaderRow extends StatelessWidget {
   }
 }
 
+class QSet extends StatelessWidget {
+  final String assessmentId;
+  final int assessmentIndex;
+  final int assessmentSetIndex;
+  final Assessment assessment; // Add assessment parameter
+
+  const QSet({
+    super.key,
+    required this.assessmentId,
+    required this.assessmentIndex,
+    required this.assessmentSetIndex,
+    required this.assessment, // Add assessment parameter
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: assessment.questions.map((question) {
+        return ListTile(
+          title: Row(
+            children: [
+              Expanded(
+                child: Text(question.questionText),
+              ),
+              IconButton(
+                icon: Icon(Icons.edit),
+                onPressed: () {
+                  _editQuestion(context, question, true);
+                },
+              ),
+            ],
+          ),
+          subtitle: Row(
+            children: [
+              Expanded(
+                child: Text(question.answer ?? ''),
+              ),
+              IconButton(
+                icon: Icon(Icons.edit),
+                onPressed: () {
+                  _editQuestion(context, question, false);
+                },
+              ),
+            ],
+          ),
+        );
+      }).toList(),
```

Comment:
Love the way this works but we need to also have an interface for updating the answer options as well as the answer itself. When you have a multiple choice question, our current configuration shows the possible answers as well as the answer.